harness_scripts/common/update_python_to_meet_requirements.sh

```diff
@@ -0,0 +1,25 @@
+#/usr/bin/env bash
```

Comment:
So this script is running in a framework env, which is our drop in env.
If we need to update python, we eventually run tests in conditions different from the environment itself, isn't it?

We should just update environment to have at least Py3.9. 
Which env doens't have Py 3.9? R Lang?